rhel8/profiles/ospp.profile

```diff
@@ -5,1220 +5,540 @@ title: 'Protection Profile for General Purpose Operating Systems'
-
-    ## Ensure the Default C Shell Umask is Set Correctly
-    - accounts_umask_etc_csh_cshrc
-
-    ## SEE ALSO: https://github.com/ComplianceAsCode/content/issues/4475
-    # - accounts_umask_etc_login_defs
-
-    #################################################################
-    ## PAM Setup
-    #################################################################
-
-    ## Disable Core Dumps for All Users
-    - disable_users_coredumps
-
-    ## Limit the Number of Concurrent Login Sessions Allowed Per User
-    - var_accounts_max_concurrent_login_sessions=10
-    - accounts_max_concurrent_login_sessions
-
-    ## RANDOM TO DO
-    #sed -i "6s/^#//" /etc/pam.d/su
-    #sed -i "8iauth        required      pam_faillock.so preauth silent " /etc/pam.d/system-auth
-    #sed -i "8iauth        required      pam_faillock.so preauth silent " /etc/pam.d/password-auth
-
-    # securetty is disabled by default in RHEL8, but it can be enabled just by editing a few files
-    - securetty_root_login_console_only
-
-    ## Limit Password Reuse
-    - accounts_password_pam_unix_remember
-    - var_password_pam_unix_remember=5
-
-    ## Set Deny For Failed Password Attempts
-    - var_accounts_passwords_pam_faillock_deny=3
-    - accounts_passwords_pam_faillock_deny
-
-    ## Set Interval For Counting Failed Password Attempts
-    - var_accounts_passwords_pam_faillock_fail_interval=900
-    - accounts_passwords_pam_faillock_interval
-
-    ## Set Lockout Time for Failed Password Attempts
-    - var_accounts_passwords_pam_faillock_unlock_time=never
-    - accounts_passwords_pam_faillock_unlock_time
-
-
-    ## Prevent Login to Accounts With Empty Password
-    - no_empty_passwords
-
-    ## TO DO: https://github.com/ComplianceAsCode/content/issues/4480
-    #sed -i 's/nullok//' /etc/pam.d/system-auth
-
-    ## TO DO: https://github.com/ComplianceAsCode/content/issues/4481
-    #sed -i 's/nullok//' /etc/pam.d/sssd-shadowutils
-
-    ## TO DO: https://github.com/ComplianceAsCode/content/issues/4479
-    ## setup tmux
-    #mv /tmp/tmux.conf /etc/tmux.conf
-    #restorecon /etc/tmux.conf
-    #sed -i 's/^fi/  if [ "$PS1" ]; then\n    [[ $TERM != "screen" ]] \&\& exec tmux\n  fi\nfi/' /etc/bashrc
-    #sed -i '/tmux$/d' /etc/shells
-
-    #################################################################
     ## Enable Automatic Software Updates
-    #################################################################
-
-    ## Configure dnf-automatic to Install Only Security Updates
+    ## SI-2 / FMT_MOF_EXT.1
+    # Configure dnf-automatic to Install Only Security Updates
     - dnf-automatic_security_updates_only
 
-    ## Configure dnf-automatic to Install Available Updates Automatically
+    # Configure dnf-automatic to Install Available Updates Automatically
     - dnf-automatic_apply_updates
 
-    ## Enable dnf-automatic Timer
-    - timer_dnf-automatic_enabled
+    # Enable dnf-automatic Timer
+    - timer_dnf-automatic_enabled
```

Comment:
Make sure there is a new line at end of file.

Reply:
Updated both RHEL7 and RHEL8 profiles with new lines.